src/test/java/core/basesyntax/service/ReadDataFileTest.java

```diff
@@ -0,0 +1,91 @@
+package core.basesyntax.service;
+
+import java.util.Map;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ReadDataFileTest {
+
+    private static String TEST_FILE = "src/test/resources/TestFile.csv";
```

Comment:
Constants should be declared as 'final' to indicate that they are not meant to be changed.